src/rpc/mempool.cpp

```diff
@@ -719,6 +719,56 @@ static RPCHelpMan getmempoolinfo()
     };
 }
 
+static RPCHelpMan importmempool()
+{
+    return RPCHelpMan{
+        "importmempool",
+        "Import the file and attempt to add its contents to the mempool.\n"
```

Comment:
Change "the file" to "a mempool.dat file" ?

Maybe reference the "savemempool" rpc as a way of updating mempool.dat with the current mempool?

Reply:
(I more just meant "what the heck type of file do you want me to give it???")